README.md

```diff
@@ -1,102 +1,139 @@
+* If you run the playbook without specifying any role variable the role will change nothing on your remote nodes
+* To execute some supported use cases you need to explicitly specify one or more of the following variables
+
+### Available tags to control and use the role
+
+## What does this role not do for you?
+
+* It does not explain how to create a good AIDE configuration that suits your requirements; that task remains for you to accomplish
 
 ## Requirements
 
-Any prerequisites that may not be covered by Ansible itself or the role should
-be mentioned here.  This includes platform dependencies not managed by the
-role, hardware requirements, external collections, etc.  There should be a
-distinction between *control node* requirements (like collections) and
-*managed node* requirements (like special hardware, platform provisioning).
+This role has no special requirements as it uses `ansible.builtin` modules
+only.
+
+## Role Variables
+
+### aide_custom_template
 
-### Collection requirements
+This variable takes a string to specify a path where the custom template for aide.conf is located.
 
-For instance, if the role depends on some collections and has a
-`meta/collection-requirements.yml` file for installing those dependencies, and
-in order to manage `rpm-ostree` systems, it should be mentioned here that the
- user should run
+To be sure that everething is correct, template needs to start with following snippet:
 
-```bash
-ansible-galaxy collection install -vv -r meta/collection-requirements.yml
+``` jinja
+{{ ansible_managed | comment }}
+{{ "system_role:aide" | comment(prefix="", postfix="") }}
 ```
 
-on the *control node* before using the role.
+Default: `null`
 
-## Role Variables
+Type: `string`
 
-A description of all input variables (i.e. variables that are defined in
-`defaults/main.yml`) for the role should go here as these form an API of the
-role.  Each variable should have its own section e.g.
+### aide_db_fetch_dir
 
-### aide_foo
+This variable takes a string to specify the directory on the Ansible Control
+Node (ACN) where the role will store the AIDE database fetched from the remote
+nodes. The default value is `files` which is expected to be a directory in the
+same directory as the playbook.
 
-This variable is required.  It is a string that lists the foo of the role.
-There is no default value.
+In case you like to store the fetched AIDE database files somewhere else you
+need to specify a different path here.
 
-### aide_bar
+Default: `files`
 
-This variable is optional.  It is a boolean that tells the role to disable bar.
-The default value is `true`.
+Type: `string
 
-Variables that are not intended as input, like variables defined in
-`vars/main.yml`, variables that are read from other roles and/or the global
-scope (ie. hostvars, group vars, etc.) can be also mentioned here but keep in
-mind that as these are probably not part of the role API they may change during
-the lifetime.
+### aide_install
```

Comment:
I think the role should always install the aide package, so this variable can be removed. Is there a scenario where users might want to set this to false?

Reply:
No, that's not the case. When you want to do just check or update for exapmple.